src/lib.rs

```diff
@@ -10,7 +10,7 @@ Spec progress as of 2020-07-04:
 ## Use as library
 ```
 fn main() -> Result<(), Box<grass::Error>> {
-    let sass = grass::from_string("a { b { color: &; } }".to_string())?;
+    let sass = grass::from_string("a { b { color: &; } }".to_string(), &grass::Options::default())?;
```

Comment:
This could use `&Default::default()`.

Reply:
I think explicitly naming the type is better.